docs/UserGuide.md

```diff
@@ -384,56 +459,147 @@ Format: `c-clear`
 
 #### 3.10 Archiving an employee : `c-archive`
 
-Archives the specified employee's contact detail from the tCheck.
+Archives the specified employee from tCheck.
 
 Format: `c-archive INDEX`
 
-* Archives the employee at the specified `INDEX`.
-* The index refers to the index number shown in the displayed `employees' contact details` panel.
-* The index **must be a positive integer** 1, 2, 3, ...​
+* Archives the employee at the specified `INDEX` in the Employee Directory inside the _Main View_.<br>
+* The `INDEX` refers to the index number shown in the current Employee Directory inside the _Main View_.<br>
+* The `INDEX` **must be a positive integer** (e.g. 1, 2, 3, ...​).<br>
+* The `INDEX` **should not be larger than** the total number of employees listed in the current 
+_Employee Directory_ pane. For example, there are 2 employees listed in the current Employee Directory inside the _Main
+ View_. The `INDEX` should be less or equal to 2.<br>
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about Employee Directory:**<br>
+
+If _Employee Directory_ currently shows a list of archived employees (archived employees are displayed with a red
+ color label, named as `[Archived]`, in front of the name), command `c-active-list` needs to be used first to see a
```

Comment:
Should it be `[Archived]` or should another format be used? Since we are using the grey block for commands only?

Reply:
Should use "" instead. Will change all `[Archived]` to "[Archived]". Thanks!